examples/baselines/act/act/detr/backbone.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+"""
+Backbone modules.
+"""
+from collections import OrderedDict
+
+import torch
+import torch.nn.functional as F
+import torchvision
+from torch import nn
+from torchvision.models._utils import IntermediateLayerGetter
+from typing import Dict, List
+
+from examples.baselines.act.act.utils import NestedTensor, is_main_process
```

Comment:
imports should be relative to act (which you pip install -e .)

Reply:
updated my code.